buildSrc/src/main/java/org/elasticsearch/gradle/test/rest/transform/warnings/InjectAllowedWarnings.java

```diff
@@ -26,21 +26,31 @@
     private static JsonNodeFactory jsonNodeFactory = JsonNodeFactory.withExactBigDecimals(false);
 
     private final List<String> allowedWarnings;
+    private final boolean isRegex;
 
     /**
      * @param allowedWarnings The allowed warnings to inject
      */
     public InjectAllowedWarnings(List<String> allowedWarnings) {
+        this(false, allowedWarnings);
+    }
+
+    /**
+     * @param isRegex true if should inject the regex variant of allowed warnings
+     * @param allowedWarnings The allowed warnings to inject
+     */
+    public InjectAllowedWarnings(boolean isRegex, List<String> allowedWarnings) {
```

Comment:
We don't seem to be tracking `isRegex` as an input in any way. Perhaps `getSkipFeatureName()` should be tracked as an input? That would work just as well here since it's conditional on that value and is no longer static.

Reply:
Thanks Mark! done on [`11482e2` (#70359)](https://github.com/elastic/elasticsearch/pull/70359/commits/11482e26c50d2b1dd5ab6bcbc55b0f404bd9b1c9)